hugo/content/contribute.md

```diff
@@ -3,83 +3,230 @@ title: Contribute
+
+Also take a look at open [issues](https://github.com/mumble-voip/mumble/issues) and [pull requests](https://github.com/mumble-voip/mumble/pulls) on GitHub, especially with the following Labels:
+
+<!-- add label for important tasks? -->
+
+- [Good first issue](https://github.com/mumble-voip/mumble/labels/good%20first%20issue)
+- [Help wanted](https://github.com/mumble-voip/mumble/labels/help%20wanted)
+- [Help needed](https://github.com/mumble-voip/mumble/labels/help-needed)
+- [Feature Request](https://github.com/mumble-voip/mumble/labels/feature-request)
+- [Bug](https://github.com/mumble-voip/mumble/labels/bug)
+
+
+#### Tips
+
+- To avoid unnecessary work and duplication, comment on issues and inform others what you plan to work on.
+- You can also create [Draft Pull Requests](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests); this way you can work on the Pull Request, while others can see the progress and give you feedback.
+
+### Additional projects
+
+#### 3rd party Interfaces
+
+Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice- or gRPC-based RPC Interfaces. You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with the Mumble server might look like, you can take a look at the example scripts in our [GitHub repository for mumble-scripts](https://github.com/mumble-voip/mumble-scripts).
 
-If you are looking for support yourself or want to provide some help to users these are the places to do so:
+#### Ice authenticators
 
-* Our [Forums](https://forums.mumble.info/)
-* IRC [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
-* Chat [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
-* Our [issue tracker](https://github.com/mumble-voip/mumble/issues)
+See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement.
 
-# Report Bugs and Issues
+#### Positional Audio plugins
 
-If you found a bug or issue you can report them in our [issue tracker](https://github.com/mumble-voip/mumble/issues). Please check if a ticket already exists for your issue before creating a new one.
+Most of our [positional audio plugins](https://wiki.mumble.info/wiki/Games#Positional_audio) regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. 
 
-# Suggest Changes and Features
+You can also add positional audio support to Games via our standard `Link` plugin. This way it will not get outdated, but you need permission to change (and access to) the Source Code of a game though (e.g. via an open license or developer permission). See {{< wiki "Link" "Linking a game to Mumble" />}}.
```

Comment:
@Krzmbrzl Check again please.

Reply:
I would remove the "though" but otherwise this LGTM

Reply:
Ok, I will remove the "though".